### DISCUSSION TITLE
Spawning a rigidbody at the same position as another

### DISCUSSION THREAD

**Discussion by unravel-dev** [xissburg/edyn#129] [Q&A]
Hi again. I am spawning 2 spheres rigidbodies at the same location. For some shapes this is defined for some it isn't and i get an assert
Example spawning 2 boxes at the same location seems to work and they seem to push eachother away which seems ok.
When spawning 2 spheres for example at the same location gets an assert here

```
create_contact_point
...
    EDYN_ASSERT(length_sqr(rp.normal) > EDYN_EPSILON);
```

i am wondering if edyn should handle this case and define some behavior instead of just bailing when the 2 bodies are at the same position.

**Comment by xissburg** [comment]
Yes it should be able to handle these situations. This can be fixed.

**Comment by xissburg** [comment]
This commit should fix it
https://github.com/xissburg/edyn/commit/3c0a3620fa6c3255d0fbd993291b55f29c12dd75
